ONSdigital/response-operations-ui
# Js test read csv

# Motivation and Context
Introduce unit testing for CSV reading in frontend

# What has changed
* Refactored `read-csv.js`
* Added tests for `read-csv.js`

# How to test?
* Review code
* Confirm `npm test` tests pass
* Ensure that reading a sample CSV still produces a summary of the CSV selected.

# Links
https://trello.com/c/kkdGNMJM
